src/lib/MsDyn.Contrib.CloneFieldDefinitions/CloneFieldsControl.cs

```diff
@@ -195,57 +197,64 @@ private void InitializeComponent()
             this.flowLayoutPanel2.Controls.Add(this.prefixOverride);
             this.flowLayoutPanel2.Controls.Add(this.button2);
             this.flowLayoutPanel2.Controls.Add(this.button3);
-            this.flowLayoutPanel2.Location = new System.Drawing.Point(3, 3);
+            this.flowLayoutPanel2.Controls.Add(this.button4);
+            this.flowLayoutPanel2.Location = new System.Drawing.Point(8, 7);
+            this.flowLayoutPanel2.Margin = new System.Windows.Forms.Padding(8, 7, 8, 7);
             this.flowLayoutPanel2.Name = "flowLayoutPanel2";
-            this.flowLayoutPanel2.Size = new System.Drawing.Size(1129, 39);
+            this.flowLayoutPanel2.Size = new System.Drawing.Size(3278, 131);
```

Comment:
Why >3000px?

Reply:
I think that might be because I am using a 4k display for development and the designer actually updated these values when I was working along with the sizes, but in short no specific reason for >3000px

Reply:
The only idea here was to enable a second row of controls in case there is an overflow